src/screens/CandidateDetail/TabBar/index.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import { View, StyleSheet } from 'react-native';
+import AboutTab from './AboutTab';
+import MatchTab from './MatchTab';
+import NewsTab from './NewsTab';
+import TabItem from './TabItem';
+
+class TabBar extends Component {
+  state = {
+    selectedTab: 'Match',
+  }
+
+  // Replace with Redux Action
+  renderSelectedView = () => {
```

Comment:
It might be easier to wrap the whole Candidate screen in a tab navigator than to write that functionality yourself. But since you've already done it, good job!

Reply:
That was actually my first plan, but I wasn't quite sure how to work out. Tried dropping in a tab navigator as a component under the candidate details section, but got a console warning about having more than one navigator. Would love to see how that might work. Could not find any react-navigator examples of a tab navigator that appears below a component like the mockup.